LSP analyze all files in project (#13150)
ethereum/solidity: Changelog.md

Compiler Features:

>  * LSP: Analyzes all files in a project by default (can be customized by setting ``'analyze-all-files-in-project'`` to ``false`` in LSP settings object).

Reviewer: We seem to have a mix of `LSP` and `Language Server` in the changelog. We should settle on one :)

>  * Language Server: Analyzes all files in a project by default (can be customized by setting ``'analyze-all-files-in-project'`` to ``false`` in LSP settings object).

Reviewer: ```suggestion
 * Language Server: Analyze all files in a project by default (can be customized by setting ``'file-load-strategy'`` to ``'directly-opened-and-on-import'`` in LSP settings object).
```
